vowpalwabbit/example_predict.cc

```diff
@@ -17,3 +16,10 @@ safe_example_predict::~safe_example_predict()
   for (size_t i=0;i<256;i++)
     feature_space[i].delete_v();
 }
+
+void safe_example_predict::clear()
+{
+	for (auto ns : indices)
+		feature_space[ns].clear();
+	indices.clear();
+}
```

Comment:
should ft_offset be reset?

Reply:
@JohnLangford the new code doesn't assume any previous state. What's your take on this?